packages/components/stories/Skeleton.js

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import talendIcons from '@talend/icons/dist/react';
 import Skeleton from '../src/Skeleton';
 import IconProvider from '../src/IconsProvider';
 
+import theme from './Skeleton.scss';
+
 const stories = storiesOf(Skeleton.displayName, module);
 if (!stories.addWithInfo) {
 	stories.addWithInfo = stories.add;
 }
 
-const icons = {
-	'talend-locked': talendIcons['talend-locked'],
-};
-
 stories
-	.addDecorator(story => <div className="col-lg-offset-2 col-lg-8">{story()}</div>)
+	.addDecorator(story => (
+		<div className="col-lg-offset-2 col-lg-8">
+			<IconProvider />
```

Comment:
All icons are stored in the snapshot this way.
With selected icons passed for the test, this was not the case.

Reply:
why do you need to test IconProvider. it's not necesseray.

Reply:
It's here to make the icons work, it's not test as itself 🤔